app.rb

```diff
@@ -16,16 +16,17 @@ def escape(memo)
   end
 end
 
-def memo_specified_by_id(params_id)
-  @connected_db.exec_params('SELECT * FROM memos WHERE id = $1;', [params_id])
+def find(params_id)
```

Comment:
```suggestion
def find(id)
```

ブラウザから渡されたパラメータだからparamsって付けたのかな？
とはいえ、findメソッドからすれば、どこから渡ってきたのかは何も気にしないはずなので、ここはidだけで良いと思いますー。

Reply:
> findメソッドからすれば、どこから渡ってきたのかは何も気にしないはずなので、ここはidだけで良いと思いますー。

了解しました！
なるほど、そのような考え方で引数名等も考えるのですね😳
かしこまりました！